src/framework/core/js/FluidViewDebugging.js

```diff
@@ -0,0 +1,649 @@
+            paneRow: "<tr class=\"flc-debug-pane-row\" id=\"%rowId\"><td class=\"fl-debug-pane-index\">%indexEl</td><td class=\"flc-debug-dom-id\">%domId</td><td class=\"flc-debug-component-id\">%componentId</td><td class=\"flc-debug-pane-grades %extraGradesClass\">%grade</td><td class=\"flc-debug-pane-line %extraTooltipClass\" id=\"%tooltipTriggerId\">%line</td></tr>",
+            indexElement: "<div class=\"flc-debug-pane-indexel\" style=\"background-color: %colour\"></div>"
+        },
+        selectors: {
+            openPaneTrigger: ".flc-debug-open-pane-trigger",
+            inspectTrigger: ".flc-debug-inspect-trigger",
+            holder: ".fl-debug-holder",
+            pane: ".fl-debug-pane",
+            paneBody: ".flc-debug-pane-body",
+            indexEl: ".flc-debug-pane-indexel",
+            row: ".flc-debug-pane-row"
+        },
+        events: {
+            onNewDocument: null,
+            onMarkupReady: null,
+            highlightClick: null
+        },
+        listeners: {
+            "onCreate.render": {
+                priority: "first",
+                funcName: "fluid.debug.browser.renderMarkup",
+                args: ["{that}", "{that}.options.markup.holder", "{that}.options.markup.pane"]
+            },
+            "onCreate.toggleTabClick": {
+                funcName: "fluid.debug.bindToggleClick",
+                args: ["{that}.dom.openPaneTrigger", "{that}.applier", "isOpen"]
+            },
+            "onCreate.toggleInspectClick": {
+                funcName: "fluid.debug.bindToggleClick",
+                args: ["{that}.dom.inspectTrigger", "{that}.applier", "isInspecting"]
+            },
+            "onCreate.bindHighlightSelection": {
+                funcName: "fluid.debug.browser.bindHighlightSelection",
+                args: ["{that}", "{that}.dom.pane"]
+            },
+            "onNewDocument.bindHover": {
+                funcName: "fluid.debug.browser.bindHover",
+                args: ["{that}", "{arguments}.0"]
+            },
+            "onNewDocument.bindHighlightClick": {
+                funcName: "fluid.debug.browser.bindHighlightClick",
+                args: ["{that}", "{arguments}.0"]
+            },
+            highlightClick: {
+                funcName: "fluid.debug.browser.highlightClick",
+                args: "{that}"
+            }
+        },
+        components: {
+            tooltips: {
+                createOnEvent: "onMarkupReady",
+                type: "fluid.tooltip",
+                container: "{browser}.dom.pane",
+                options: {
+                    items: ".flc-debug-tooltip-trigger",
+                    styles: {
+                        tooltip: "fl-debug-tooltip"
+                    },
+                    position: {
```

Comment:
Typo - should read "center" but in fact was being ignored due to this being the default in any case - http://api.jqueryui.com/position/